src/tests/test_sexual_behaviour.py

```diff
@@ -52,7 +57,25 @@ def check_num_partners(row):
 def test_num_partners():
     """Check that number of partners are reasonable"""
     pop_data = Population(size=1000, start_date=date(1989, 1, 1)).data
-    print(pop_data)
+    assert(any(pop_data["num_partners"] > 0))
     # Check the num_partners column
     checks = pop_data.apply(check_num_partners, axis=1)
     assert np.all(checks)
+
+
+def test_sex_behaviour_groupings():
+    """Check that people are assigned to all sex behaviour groups!"""
+    pop_data = Population(size=1000, start_date=date(1989, 1, 1)).data
+    for sex in SexType:
+        for group in SexBehaviours[sex]:
+            index = selector(pop_data, sex=(operator.eq, sex), sex_behaviour=(operator.eq, group))
+            assert any(index)
+
+
+def test_behaviour_updates():
```

Comment:
Could add a docstring to clarify the intent here, and possibly update it if the test evolves.

```suggestion
def test_behaviour_updates():
    """Check that at least one behaviour group changes with time."""
```